frontend-react/src/components/MessageIdSearch/MessageIdSearch.tsx

```diff
@@ -0,0 +1,137 @@
+import React, { FormEvent, useState } from "react";
+import { Label, Search } from "@trussworks/react-uswds";
+import Table, { TableConfig } from "../../components/Table/Table";
+import { withCatchAndSuspense } from "../RSErrorBoundary";
+import Spinner from "../Spinner";
+
+const MOCK_MESSAGE_ID_DATA = [
+    {
+        id: 1,
+        messageId: "12-234567",
+        sender: "somebody 1",
+        submittedDate: "09/28/2022",
+        reportId: "29038fca-e521-4af8-82ac-6b9fafd0fd58",
+    },
+    {
+        id: 2,
+        messageId: "12-234567",
+        sender: "somebody 2",
+        submittedDate: "09/29/2022",
+        reportId: "86c4c66f-3d99-4845-8bea-111210c05e63",
+    },
+    {
+        id: 3,
+        messageId: "12-234567",
+        sender: "somebody 3",
+        submittedDate: "09/30/2022",
+        reportId: "26a37945-ed12-4578-b4f6-203e8b9d62ce",
+    },
+];
+
+// TODO: move this interface into the resources directory
+interface MessageIdResource {
```

Comment:
should we call this `MessageSeachResource`? or maybe `MessageListResource`? `MessageSeachResultResource`? i feel having the word `Id` makes it confusing as the table is a list of Messages (not just ids)

Reply:
Will update